index.html

```diff
@@ -250,12 +296,12 @@ <h2 class="contact-form-h1 poppins">Contact Me </h2>
           <form action="https://formspree.io/f/mnqrnjol" method="POST">
             <ul class="contact-form-list">
               <li>
-                <input type="text" id="firstName" name="name" class="contact-name poppins" placeholder="Richard"
+                <input type="text" id="firstName" name="name" class="contact-name poppins" placeholder="Richard Sage"
                   required maxlength="30" />
               </li>
               <li>
                 <input type="email" id="email" name="email" class="contact-email poppins"
-                  placeholder="richard@gmail.com" required />
+                  placeholder="sage@rico.com" required />
```

Comment:
Kindly modify the contact form section to match your chosen Figma design. Check the comment above for more reason you should follow a design when given.

Your design:
![image](https://user-images.githubusercontent.com/20554583/186802653-2be5d1e7-ccf0-4e02-879d-e6b17c0eb77b.png)


Figma design:
![image](https://user-images.githubusercontent.com/20554583/186802978-a1f23878-fe91-4f11-bd57-d1472da9d945.png)